### PR TITLE
Beautification, show-lists command using cache, flexible cache file format, and card-details command

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Updates:
- A beautification script has been run on every file
- Update commands to follow the same creation format
- Add output functions for every allowed color and format
- output.emphasis is now output.underline (with backwards compatibility)
- Update package.json information
- Rewrite cache file format to provide flexibility for adding new information for boards, organizations, and lists
- Add user cache with a reserved entry for the end user
- Add show-lists command (syjust created one, however it does not use the cache file where the data is already stored)
- Add card-details command (display information about a card from it’s ID, short ID, or short URL)
- Minor updates

Known issues with card-details:
- Comments cannot be viewed, as the API doesn’t seem to provide a way
to easily view comments
- Uses colors, a lot

### CODE REVIEW

**Comment by mheap** [comment]
@vinyldarkscratch I will be looking at this over the next week or so. It's a large PR and christmas is a busy time. I haven't forgotten about you

File: src/commands/show-lists.js [hunk 1]
```diff
@@ -2,70 +2,105 @@
...
 
+        boardIds.forEach(function(boardId) {
+            var listOfLists = [];
+            _.each(translator.cache.translations.lists, function(oneList, listId) {
+                if (listId != "undefined" && (oneList["board"] == boardId)) {
+                    // oneList: [ boardId, listName ]
+                    listOfLists.splice(
+                        _.sortedIndex(listOfLists, {
+                            id: listId,
+                            name: oneList["name"]
```
Comment (mheap) [inline src/commands/show-lists.js]: Should this have `else { output.underline(options.board }` ?

Comment (vinyldarkscratch) [inline src/commands/show-lists.js]: I'm not following what you mean, @mheap?  Would you be willing to further explain?  (I also just woke up, my morning brain may have something to do with that... XD)

**Comment by vinyldarkscratch** [comment]
Thank you very much, and we're glad to have given you interest in this once again!  😄

**Pull request merged**